src/main/java/com/epam/service/UserDetailsServiceImpl.java

```diff
@@ -22,7 +23,7 @@
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
         User user = userService.getUser("user");
         Set<GrantedAuthority> roles = new HashSet<>();
-        roles.add(new SimpleGrantedAuthority(user.getRole().getEnumrole()));
+        roles.add(new SimpleGrantedAuthority (UserRoleEnum.USER.getEnumrole()));
```

Comment:
Why u hardcode `userRole` here if it should depend on the user from DB?

Reply:
Because branch dev hasnt DB component, it doesnt matter what inlcluded here

Reply:
Anyway it will be changed, when we ll merge our dev with DB branch